File: android_client/app/src/main/java/com/example/minervaApp/Student.java
package com.example.minervaApp;

import java.io.Serializable;

public class Student implements Serializable {
    private int id;
    private String name;
    private String surname;

    public Student(int id, String name, String surname) {
        this.id = id;
        this.name = name;
        this.surname = surname;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getSurname(){ return surname; }

    public String toString() {
        return name + "(" + id + ")";
    }
}

File: client/coms/student.py
class Student:
    """Class representing a student"""
    def __init__(self, u_id, name, surname, uid):
        self._id = u_id
        self._name = name
        self._surname = surname
        self._uid = uid

    def get_id(self):
        return self._id
    
    def get_name(self):
        return self._name
    
    def get_surname(self):
        return self._surname

    def get_uid(self):
        return self._uid

    def __str__(self):
        """String representation of the class."""
        return f'({self._id},{self._name} {self._surname},{self._uid})'